src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,73 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "Orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    @JoinColumn(name = "ticket_id")
```

Comment:
The `@OneToMany` annotation on the `tickets` field should typically use `mappedBy` to indicate the owning side of the relationship, rather than `@JoinColumn`. Consider updating this to ensure proper bidirectional mapping.